src/containers/flows/AcceptBid/components/Success/Success.tsx

```diff
@@ -0,0 +1,76 @@
+//- Components Imports
```

Comment:
nit: could probably make a more generic component that handles `Success.tsx` and `Details.tsx` to reduce code duplication